AudibleApi/ApiUnauthenticated [partial].cs

```diff
@@ -0,0 +1,46 @@
+﻿using Dinah.Core.Net.Http;
+using System;
+using System.Linq;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace AudibleApi
+{
+	public partial class ApiUnauthenticated
+	{
+		public virtual bool IsAuthenticated => false;
+		public IHttpClientSharer Sharer { get; }
+		protected Locale _locale { get; }
```

Comment:
Moving `_locale` to here is fine. Now that it's protected if it were me I'd change it to `Locale`. Nothing I'm insisting on though.

Reply:
Done.